.gitignore

```diff
@@ -8,3 +8,4 @@
 /gpii/node_modules/gpii-oauth2/gpii-oauth2-utilities/node_modules/
 
 npm-debug.log
+*.bak
```

Comment:
Oh, that comes from Notepad++: it creates .bak files (backups) for each file you modify. Should I remove it?